spec/factories/other_advice_method.rb

```diff
@@ -1,5 +1,6 @@
 FactoryGirl.define do
   factory :other_advice_method do
     name { Faker::Lorem.sentence }
+    sequence(:cy_name) { |n| "Dull gyngor arall #{n}" }
```

Comment:
I added the sequence so I could uniquely identify the options by name in the specs. Also, the `name` here isn't a sequence so it is somewhat contradictory.